02.calendar/calendar.js

```diff
@@ -0,0 +1,31 @@
+import { DateTime } from "luxon";
+import minimist from "minimist";
+
+const argv = minimist(process.argv.slice(2));
+const today = DateTime.now();
+
+const year = parseInt(argv.y) || today.year;
+const month = parseInt(argv.m) || today.month;
+
+const firstDate = DateTime.fromObject({
+  year: year,
+  month: month,
+});
+const firstDay = firstDate.weekday;
+
+console.log(`      ${firstDate.month}月 ${firstDate.year}`);
+console.log("日 月 火 水 木 金 土");
+if (firstDate.weekday !== 7) {
+  process.stdout.write("   ".repeat(firstDay));
+}
+for (let day = 1; day <= firstDate.daysInMonth; day++) {
+  const datetime = firstDate.set({ day });
+  let dayString = String(day).padStart(2, " ");
+  if (datetime.weekday === 6) {
+    dayString += "\n";
+  } else {
+    dayString += " ";
+  }
```

Comment:
```suggestion
  const dayString = String(day).padStart(2, " ");
  const separator = datetime.weekday === 6 ? "\n" : " ";
```

JSはなるべくconstで宣言したいので、こんなふうにするとよいかも？

Reply:
letよりconstの方が、再代入を考えないで良い分意図が明確化されるということですね。
ご提案いただいた形に修正しました。
[🐛 fix: 変数名変更](https://github.com/reckyy/js-practices/pull/2/commits/7fe83c3e5610a48aae1aac5bf9b0e59d707ed97d)